### PR TITLE
feat: get the list of organizations

### PR DESCRIPTION
**Pull request by 0xff-dev**
feat: get the list of organizations.
https://github.com/bestchains/bc-cli/issues/2

### CODE REVIEW

**Comment by 0xff-dev** [comment]
https://github.com/bestchains/bc-cli/issues/19#issuecomment-1543651389

File: pkg/org/get.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+				fmt.Fprintln(option.ErrOut, err)
+				return err
+			}
+			_ = p.PrintObj(obj, option.Out)
+			return nil
+		},
+	}
+
+	defaultPrintFlag.AddFlags(cmd)
+	cmd.Flags().StringVar(&kubeconfig, "kubeconfig", fmt.Sprintf("%s/%s/%s", homedir.HomeDir(), clientcmd.RecommendedHomeDir, clientcmd.RecommendedFileName), "kube config path")
```
Comment (bjwswang) [inline pkg/org/get.go:119]: why we still need this flag? `kubeconfig`

File: pkg/common/config.go [hunk 2]
```diff
@@ -21,9 +21,55 @@ package common
...
-	Auth AuthConfig `mapstructure:"auth"`
-	Saas SaasConfig `mapstructure:"saas"`
+	Auth    AuthConfig    `mapstructure:"auth"`
+	Saas    SaasConfig    `mapstructure:"saas"`
+	Cluster ClusterConfig `mapstructure:"cluster"`
+}
+
+// copy from "k8s.io/client-go/tools/clientcmd/api"
+// Cluster contains information about how to communicate with a kubernetes cluster
+type ClusterConfig struct {
```
Comment (bjwswang) [inline pkg/common/config.go:36]: seems this field is useless ?

Comment (0xff-dev) [reply pkg/common/config.go:36]: It is the content of the cluster section of the configuration file.

Comment (0xff-dev) [reply pkg/common/config.go:36]: Not used now, but should be used later.

File: pkg/common/ibp.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+const (
+	IBPGroup   = "ibp.com"
+	IBPVersion = "v1beta1"
+
+	OrganizationResource = "organizations"
+	FederationResource   = "federations"
+)
+
+func InKubeGetter() (*clientcmdapi.Config, error) {
```
Comment (bjwswang) [inline pkg/common/ibp.go:32]: so we better pass `Config` to this func

File: pkg/common/config.go (hunk 2 above)
Comment (bjwswang) [reply pkg/common/config.go:36]: > Not used now, but should be used later.

1. Read from `config.yaml` and merge with viper
2. use this `Config` for initialization
- kube client initalization 
- ...

**Review by bjwswang — APPROVED** [review]
/lgtm

**Review by Lanture1064 — APPROVED** [review]

**Comment by bjwswang** [comment]
@dayuy this a an example on how to `get orgs` from `k8s cluster`.

**Pull request merged**